core/modules/wmgr/python/dbMgr.py

```diff
@@ -604,8 +639,91 @@ def locadData(dbName, tblName, chunkId=None):
     _validateDbName(dbName)
     _validateTableName(tblName)
 
-    _log.error('method not implemented yet')
-    raise ExceptionResponse(501, "NotImplemented", "Data loading operations are not implemented yet")
+    # check that table exists
+    dbConn = Config.instance().dbConn()
+    if not dbConn.tableExists(tblName, dbName):
+        raise ExceptionResponse(404, "TableMissing", "Table %s.%s does not exist" % (dbName, tblName))
```

Comment:
Do you mean `chunkId is not None`? If `chunkId` is `None` then we load data into `tblName` and it must exist. If `chunkId` is not `None` then we load data into chunk table which has different name constructed below, but assumption here is that "original" table still exist as its schema is used to create chunk tables. The check is not critical, mostly to confirm that we are in a sane environment.